pkg/providers/amifamily/bootstrap/bottlerocket.go

```diff
@@ -79,6 +81,16 @@ func (b Bottlerocket) Script() (string, error) {
 	for _, taint := range b.Taints {
 		s.Settings.Kubernetes.NodeTaints[taint.Key] = append(s.Settings.Kubernetes.NodeTaints[taint.Key], fmt.Sprintf("%s:%s", taint.Value, taint.Effect))
 	}
+
+	if lo.FromPtr(b.InstanceStorePolicy) == v1.InstanceStorePolicyRAID0 {
+		s.Settings.BootstrapCommands = map[string]BootstrapCommand{
+			"000-mount-instance-storage": {
```

Comment:
It's not strongly typed anywhere. It's just the name that you assign to the command. I don't think that it matters _a ton_ what we call this since it's just going to appear in the userData and that's it.